portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 Google LLC
```

Comment:
This commit has a lot going on. This deletion logic seems like it could be broken out into a separate commit and is independent from the filtering and sorting logic.